### PR TITLE
added esConsumes to modules in EventFilter/1GlobalTriggerRawToDigi

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 6 modules
#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This change was done under my supervision.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
please test

File: EventFilter/L1GlobalTriggerRawToDigi/src/L1GtTriggerMenuLiteProducer.cc [hunk 1]
```diff
@@ -62,6 +48,12 @@ L1GtTriggerMenuLiteProducer::L1GtTriggerMenuLiteProducer(const edm::ParameterSet
       m_l1GtPfAlgoCacheID(0ULL),
       m_l1GtPfTechCacheID(0ULL),
 
+      m_l1GtStableParamToken(esConsumes<L1GtStableParameters, L1GtStableParametersRcd, edm::Transition::BeginRun>()),
+      m_l1GtPfAlgoToken(esConsumes<L1GtPrescaleFactors, L1GtPrescaleFactorsAlgoTrigRcd, edm::Transition::BeginRun>()),
+      m_l1GtPfTechToken(esConsumes<L1GtPrescaleFactors, L1GtPrescaleFactorsTechTrigRcd, edm::Transition::BeginRun>()),
+      m_l1GtTmAlgoToken(esConsumes<L1GtTriggerMask, L1GtTriggerMaskAlgoTrigRcd, edm::Transition::BeginRun>()),
+      m_l1GtTmTechToken(esConsumes<L1GtTriggerMask, L1GtTriggerMaskTechTrigRcd, edm::Transition::BeginRun>()),
+      m_l1GtMenuToken(esConsumes<L1GtTriggerMenu, L1GtTriggerMaskTechTrigRcd, edm::Transition::BeginRun>()),
```
Comment (Dr15Jones) [inline EventFilter/L1GlobalTriggerRawToDigi/src/L1GtTriggerMenuLiteProducer.cc:56]: This should be
```C++
m_l1GtMenuToken(esConsumes<L1GtTriggerMenu, L1GtTriggerMenuRcd, edm::Transition::BeginRun>()),
```

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
ping

**Comment by rekovic** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**